sanity-io/sanity
# Fix: prevent flash of "Untitled" when loading a document w/ preview

### Description
This PR fixes the quick flash of a Document's title as "Untitled" if the document has a preview (e.g. the movie database template) by adding a ready check before showing the Document pane.

### What to review

Verify that there is not a case where we should show the Document pane, but the Document is not ready.

### Testing

1. `npm create sanity@latest`
2. Choose the movie database template, typescript
3. `npm run dev`
4. Visit http://localhost:3333/structure/movie and click on the various movies in the list. You should no longer seewill see "Untitled" flash as their title.

### Notes for release
